frontend/src/pushapi.js

```diff
@@ -21,49 +17,16 @@ angular.module('clouway-push', [])
     };
 
     /**
-     * Set a method to call for opening of connection.
-     * The method must return a promise.
+     * Sets the backend url connection.
      *
-     * @param {function(subscriber)} method method that will be called for opening of connection.
+     * @param url the new url
      * @returns {*} the provider instance for chaining purposes.
      */
-    this.openConnectionMethod = function (method) {
-      this.connectionMethods.connect = method;
+    this.backendServiceUrl = function(url) {
```

Comment:
I'm a bit confused: 

karma config says that files src/testing/ is not loaded for testing: 
```
/**
     * This is the list of file patterns to load into the browser during testing.
     */
    files: [
      //* injector:js *//
      //* endinjector *//
      'src/**/*.js'
    ],
    exclude: [
      'src/testing/**/*.js'
    ],

```

but pushapi-mocks.js was in src/testing folder?